centos/Dockerfile

```diff
@@ -48,11 +48,15 @@ RUN set -ex \
   && sed -i 's/^assume_yes=""/assume_yes="-y"/' install.sh \
   # TODO: remove once launching agent using `service` has been handled in the install script
   && sed -i '/^# Unconditionally stop the agent service/,$d' install.sh \
-  && sh ./install.sh -y
+  && sh ./install.sh -y \
+  # cleanup sensitive nginx-plus data
+  && rm /etc/ssl/nginx/nginx-repo.* \
+  && rm /etc/yum.repos.d/nginx-plus-7.4.repo \
+  && gpg --batch --delete-keys "$NGINX_GPGKEY"
 
 # Forward request logs to Docker log collector
 RUN ln -sf /dev/stdout /var/log/nginx-controller/agent.log \
-  && ln -sf /dev/stderr /var/log/nginx/error.log 
+  && ln -sf /dev/stderr /var/log/nginx/error.log
```

Comment:
I think `SPACE` added by mistake.

Reply:
this change is **removing** the unnecessary space, not adding it